manifest.json

```
@@ -0,0 +1,81 @@
+{
+  "name": "permission.site",
+  "short_name": "permission.site",
+  "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+  "start_url": "./?utm_source=web_app_manifest",
```

Any reason we need to use URLs with web tracking parameters?
```suggestion
  "start_url": "./",
```
(If there is a good reason, that should probably be documented.)